### PR TITLE
Angor Vs. Traditional Crowdfunding

### PR DESCRIPTION
**Pull request by Danielogenna**
An article comparing Angor Vs. Traditional Crowdfunding.

### CODE REVIEW

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+title: "Angor vs. Traditional Crowdfunding: A Comparative Analysis"
+published: 2024-05-25
+description: "This article compares Angor to traditional crowdfunding services, outlining key differences, advantages, and potential challenges."
+image: "./Angor Vs. Traditional Crowdfunding.svg"
+tags: ['Decentralized Crowdfunding', 'Bitcoin Crowdfunding' ,'Nostr','Fundraising']
+category: ['Finance and Investment', 'Crowdfunding' ,'Blockchain and Decentralization','Technology']
+draft: false
+---
+
+Crowdfunding has revolutionized the way individuals and businesses raise capital. Traditional platforms like Kickstarter, Indiegogo, and GoFundMe have paved the way for countless projects, from innovative gadgets to humanitarian causes. However, as technology evolves, so do the methods of crowdfunding. Enter Angor, a decentralized crowdfunding platform leveraging Bitcoin and Nostr for enhanced security and transparency.
```
Comment (dangershony) [inline src/content/posts/Angor vs. Traditional Crowdfunding/index.md:11]: you can also add CrowedCube

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+### 1. Platform Structure
+
+#### Traditional Crowdfunding Platforms:
+Centralized: Platforms like Kickstarter and Indiegogo operate as centralized entities, managing all transactions and interactions between creators and backers.
+
+Middleman Dependency: These platforms act as intermediaries, which means they have control over the fund distribution and project approval processes.
+
+#### Angor:
+Decentralized: Angor operates on a decentralized model using Bitcoin for financial transactions and Nostr for communication. This removes the need for a central authority, giving more control to users.
```
Comment (dangershony) [inline src/content/posts/Angor vs. Traditional Crowdfunding/index.md:25]: users => investors

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Traditional Crowdfunding Platforms:
+Platform Fees: Traditional platforms charge a percentage of the funds raised, typically ranging from 5% to 10%, in addition to payment processing fees.
+
+Hidden Costs: Additional costs may include promotional fees or charges for premium features
+
+#### Angor:
+Lower Fees: As a decentralized platform, Angor can offer lower fees by eliminating the need for a central authority and reducing operational costs.
+
+Transparency in Costs: Any fees associated with Bitcoin transactions or Nostr usage are typically lower and more transparent.
```
Comment (dangershony) [inline src/content/posts/Angor vs. Traditional Crowdfunding/index.md:63]: We charge 1%

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Angor:
+Nostr Integration: Angor uses Nostr, a decentralized communication protocol, for project updates. This ensures that updates are transparent and directly accessible to investors.
+
+Direct Communication: Project founders can post updates directly, maintaining accountability and fostering trust through regular, detailed communication.
+
+### 4. Costs and Fees
+
+#### Traditional Crowdfunding Platforms:
+Platform Fees: Traditional platforms charge a percentage of the funds raised, typically ranging from 5% to 10%, in addition to payment processing fees.
```
Comment (dangershony) [inline src/content/posts/Angor vs. Traditional Crowdfunding/index.md:56]: 5% to 15% actually

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Traditional Crowdfunding Platforms:
+Regional Restrictions: Traditional platforms often have regional restrictions due to banking and regulatory issues, limiting global participation.
+
+Currency Limitations: Transactions are usually conducted in specific currencies, which may not be convenient for international backers.
+
+#### Angor:
+Borderless Transactions: Utilizing Bitcoin allows for seamless global transactions, enabling anyone with internet access to participate.
+
+Cryptocurrency Advantage: The use of cryptocurrency bypasses traditional banking systems, offering a more inclusive and accessible platform for global users.
```
Comment (dangershony) [inline src/content/posts/Angor vs. Traditional Crowdfunding/index.md:75]: bitcoin not cryptocurrency

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md (hunk 1 above)
Comment (miwahn) [reply src/content/posts/Angor vs. Traditional Crowdfunding/index.md]: - Yes add Crowdcube and Seedrs maybe (both Equity platforms)
- Would also add the fact that crowdfunding offered new investment opportunities for individuals (can access private equities and diversify)

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md (hunk 2 above)
Comment (miwahn) [reply src/content/posts/Angor vs. Traditional Crowdfunding/index.md]: Would make it more explicit: giving more controls to investors and fundraisers (the latter also depend on current platforms to distribute funds etc)

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md (hunk 1 above)
Comment (Danielogenna) [reply src/content/posts/Angor vs. Traditional Crowdfunding/index.md]: Updated it

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md (hunk 3 above)
Comment (miwahn) [reply src/content/posts/Angor vs. Traditional Crowdfunding/index.md:63]: - Only 1%? I'd say 2-3% is already very cheap vs currrent platforms
- The second point feels a bit week to me. 1) it's repeating the 'lower cost' argument above and 2) more transparent than what exactly? Transaction costs in fiat are also transparent. Can make a point about the fee structure being SIMPLE though (currrent platforms have multiple layers of fees, although they could also make it easier IF they wanted to...)

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md (hunk 4 above)
Comment (miwahn) [reply src/content/posts/Angor vs. Traditional Crowdfunding/index.md]: - I'd keep 5-10% as a 'typical range'
- Would add the fact they are sometimes charging other fees as well (set up fees, transfer fees etc) -> goes back to the previous point and fee structure being more simple

File: src/content/posts/Angor vs. Traditional Crowdfunding/index.md (hunk 5 above)
Comment (miwahn) [reply src/content/posts/Angor vs. Traditional Crowdfunding/index.md]: - "offering" -> "and offers"

**Comment by miladsoft** [comment]
image
When writing file names, use - instead of spaces and try to use only lowercase English letters. Please refer to my posts.

**Pull request closed without merging**